helm/zenko/charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -31,10 +31,8 @@ spec:
 {{- if .Values.global.mongodbBackend.enabled }}
             - name: S3METADATA
               value: "mongodb"
-            - name: MONGODB_HOST
-              value: "{{ .Release.Name }}-mongodb-replicaset"
-            - name: MONGODB_PORT
-              value: "{{ .Values.global.mongodbBackend.port }}"
+            - name: MONGODB_HOSTS
+              value: "{{ .Release.Name }}-mongodb-replicaset-0.zenko-mongodb-replicaset.default.svc.cluster.local:27017,{{ .Release.Name }}-mongodb-replicaset-1.zenko-mongodb-replicaset.default.svc.cluster.local:27017,{{ .Release.Name }}-mongodb-replicaset-2.zenko-mongodb-replicaset.default.svc.cluster.local:27017"
```

Comment:
change any hardcoded "zenko" reference to {{ .Release.Name }} because it is deployment specific

Reply:
Same goes with the "default" and "cluster" (each should have their own variable to use)

Reply:
You don't need to suffix the namespace and svc.cluster.local etc. Resolving should happen in the 'right' domain already.